skll/metrics.py

```diff
@@ -270,11 +273,11 @@ def register_custom_metric(custom_metric_path: PathOrStr, custom_metric_name: st
 
     # once we know that the module name is okay, we need to make sure
     # that the metric function name is also okay
-    if custom_metric_name in _SCORERS:
+    if custom_metric_name in get_scorer_names() or custom_metric_name in _CUSTOM_METRICS:
```

Comment:
Yeah, this would only happen if `register_custom_metric()` gets called and that will only happen in an experiment if (from `experiments.py`):

```
            if (
                metric_name not in get_scorer_names()
                and metric_name not in _CUSTOM_METRICS
                and metric_name is not None
            ):
```

So, overriding a scikit-learn metric name in a config file won't trigger this. However, if I use the same `custom.py` as in the docs and rename `f075` to `f1_score_micro`, then calling `register_custom_metric()` will raise this error.

Reply:
May be it's worth adding a warning the docs that overriding scikit-learn metrics will not work whether using experiments or the API.